framework/resource_router.go

```diff
@@ -36,9 +38,6 @@ func (r *ResourceRouter) ResourceSet(obj interface{}) (*ResourceSet, error) {
 		}
 	}
 
-	if len(found) == 0 {
-		return nil, microerror.Maskf(executionFailedError, "handling resource set not found")
-	}
```

Comment:
Wouldn't it be good to log that the object won't be handled by the framework?

Reply:
I asked for opinions about this in the PR description. I can add a log for this. I would put a debug level log here.

Reply:
👍

Reply:
Please have another look.

Reply:
We need to configure a logger now to all resource sets.